### PR TITLE
Improve modinst system test coverage to 100%

### PR DESCRIPTION
**Pull request by Fladolcetta**
[x ] This contribution adheres to [CONTRIBUTING.md]

### What does this Pull Request accomplish?

This updates some of the generated library files so that system tests are possible to get to 100%.
There are some error handling functions that should never happen on a valid system, so I am excluding them from the test coverage results.
This also creates an html document for better coverage analysis.

### Why should this Pull Request be merged?
Improves coverages
Improves ability to analyze coverage

### What testing has been done?

Built and ran unit and system tests.

### CODE REVIEW

File: docs/nidmm/examples.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Examples
 nidmm_measurement.py
 --------------------
 
-.. literalinclude:: ../../src/nidmm/examples/nidmm_measurement.py
+.. literalinclude:: ..\..\src\nidmm\examples\nidmm_measurement.py
```
Comment (texasaggie97) [inline docs/nidmm/examples.rst]: Undo these changes.

**Comment by Fladolcetta** [comment]
> This is approved once you revert the changes to the rst files that changed '/' to ''

Done!
